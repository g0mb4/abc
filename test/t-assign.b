main() {
    extrn printn, putchar;
    auto a, b;

    /* 2 */
    a = 2;
    printn(a, 10);
    putchar('*n');

    /* 33 */
    a = b = 3;
    printn(a, 10);
    printn(b, 10);
    putchar('*n');

    /* 5 */
    a =+ 2;
    printn(a, 10);
    putchar('*n');

    /* 4 */
    a =- 1;
    printn(a, 10);
    putchar('*n');

    /* 12 */
    a =* 3;
    printn(a, 10);
    putchar('*n');

    /* 3 */
    a =/ 4;
    printn(a, 10);
    putchar('*n');

    /* 1 */
    b = 21;
    b =% 4;
    printn(b, 10);
    putchar('*n');
}
