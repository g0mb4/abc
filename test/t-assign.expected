2
33
5
4
12
3
1
